site/content/docs/user-guide/setup/install.md

```diff
@@ -52,10 +52,10 @@ Krew self-hosts).
       set -x; set temp_dir (mktemp -d); cd "$temp_dir" &&
       set OS (uname | tr '[:upper:]' '[:lower:]') &&
       set ARCH (uname -m | sed -e 's/x86_64/amd64/' -e 's/\(arm\)\(64\)\?.*/\1\2/' -e 's/aarch64$/arm64/') &&
-      curl -fsSLO "https://github.com/kubernetes-sigs/krew/releases/latest/download/krew.tar.gz" &&
-      tar zxvf krew.tar.gz &&
       set KREWNAME krew-$OS"_"$ARCH &&
```

Comment:
```suggestion
      set KREW krew-$OS"_"$ARCH &&
```

Aha, so that's where `KREWNAME` came from (see other comment above). It's good to use the same name. I have a slight preference for `KREW` though.